source/mainnet/net/guides/deciding-wallet.rst

```diff
@@ -1,32 +1,34 @@
+Before you decide which wallet to use, it's important to know that you can’t import identities and accounts between the Mobile Wallet, the |bw|, and the Desktop Wallet. This is because they handle private keys in different ways as :ref:`described later<store-private-keys>` in this topic. It’s therefore essential that you choose between the wallets before you start creating identities and accounts.
 
-.. warning:: You can't exchange identities and accounts between the Mobile Wallet and the Desktop Wallet. You can, however, send CCD from one wallet to another.
+.. warning:: You can't exchange identities and accounts between the Mobile Wallet, the |bw|, and the Desktop Wallet. You can, however, send CCD from one wallet to another.
 
 Which wallet is best for you?
 =============================
 
-The wallets work on two different environments: the Desktop Wallet on a computer and the Mobile Wallet on a phone. The Mobile Wallet is more accessible and easier to use on a day-to-day basis whereas the Desktop Wallet has an extra layer of security in the form of a Ledger device.
+The wallets work on different environments: the Desktop Wallet on a computer, the Mobile Wallet on a phone, and the Browser Wallet from any device with an internet browser. The Mobile Wallet is more accessible and easier to use on a day-to-day basis whereas the Desktop Wallet has an extra layer of security in the form of a Ledger device.
+
+- Desktop Wallet: Choose this wallet if you want extra security in the form of a :ref:`Ledger<install-Ledger-app>` hardware device, or want to create multi-signature transactions. If you plan to become a baker, Concordium recommends the Desktop Wallet because of the extra security of the Ledger hardware device.
 
-- **Desktop Wallet**: Choose this wallet if you want extra security in the form of a :ref:`Ledger<install-Ledger-app>` hardware device, or want to create multi-signature transactions. If you plan to become a baker, Concordium recommends the Desktop Wallet because of the extra security of the Ledger hardware device.
+- Mobile Wallet: Choose this wallet if you want to be able to access your accounts on the go and only need to create standard (single signature) transactions. Also, if you use the Mobile Wallet, you don’t have to run a node (unless you are a baker), and you don’t need a Ledger hardware device.
 
-- **Mobile Wallet**: Choose this wallet if you want to be able to access your accounts on the go and only need to create standard (single signature) transactions. Also, if you use the Mobile Wallet, you don’t have to run a node (unless you are a baker), and you don’t need a Ledger hardware device.
+- |bw|: Choose this wallet if you want to be able to access your accounts on the go from any computer or device and you only need to create standard (single signature) transactions. This wallet is also for developers creating dApps to connect to the Concordium blockchain.
```

Comment:
I would argue that it is also for people using dApps. (Or who will be using dApps, I guess)